css/chat.css

```
@@ -181,8 +181,7 @@ html.no-supports-container.native body,
   transition: opacity 0.1s ease-out, transform 0.2s ease-out, padding-top 0.2s ease-out;
 }
 
-.fl-with-bottom-menu .chat-holder .chat-list,
-.fl-with-bottom-menu .chat-area
+.fl-with-bottom-menu .chat-holder .chat-list
```

The `{` shouldn't be on the next line.

```suggestion
.fl-with-bottom-menu .chat-holder .chat-list {
```